.github/workflows/release.yml

```diff
@@ -64,10 +69,30 @@ jobs:
 
             - uses: ./.github/actions/teardown
 
+    get-image-version:
+        # get image version from onboarding-enabler-nodejs/package.json as set in release job
+        runs-on: ubuntu-latest
+        needs:
+            - release # Ensure build and binary release work properly, meaning version is set appropriately
+        outputs:
+            imageVersion: ${{ steps.getImageVersion.outputs.imageVersion }}
+        steps:
+            - uses: actions/checkout@v2
+              with:
+                  ref: ${{ github.head_ref }}
+
+            - id: getImageVersion
+              run: |
+                  version=$(cat onboarding-enabler-nodejs/package.json | awk '/"version":/' | awk -F: '{print$2;}' | awk -F\" '{print$2;}')
+                  echo "::set-output name=imageVersion::$version"
+
     release-images:
+        needs:
+            - get-image-version
         uses: zowe/api-layer/.github/workflows/build-conformant-images.yml@master
         with:
             release: true
+            version: ${{ needs.get-image-version.outputs.imageVersion }}
```

Comment:
where do you assign imageVersion?

Reply:
@achmelo in the `get-image-version` job, it defines an output of `imageVersion` on line 78, which is set on line 87.

Reply:
now I see that, sorry and thank you